config/v3_0_experimental/types/file.go

```diff
@@ -42,6 +35,10 @@ func (f File) ValidateMode() report.Report {
 	return r
 }
 
+func (fc FileContents) Key() string {
```

Comment:
Should this `Key()` definition be moved into the next commit (`config/v3_0_exp/types: implement Key() for types`)?